Purchasing.Mvc/bundleConfig.json

```diff
@@ -0,0 +1,132 @@
+[
```

Comment:
general question: does the bundleConfig still work/is still being used in net6 apps?